acms-react-app/src/Pages/Admin_Pages/CustomerDetails_Page/CustomerDetails.js

```diff
@@ -63,6 +66,29 @@ function CustomerDetails() {
       });
   }
 
+  function confirmDelete() {
+    setShowConfirmation(true);
+  }
+
+  function cancelDelete() {
+    setShowConfirmation(false);
+  }
+
+  function executeDelete() {
+    axios.delete(`http://localhost:8080/api/v1/customers/${customerId}`)
+    .then(res => {
+      if(res.status === 204) {
+        alert("Customer has been deleted!")
+        setShowConfirmation(false);
+        navigate("/admin/customers")
+      }
+    })
+    .catch(err =>{
+      console.error("Error deleting customer:", err);
+      setShowConfirmation(false);
+    })
+  }
+
```

Comment:
Good stuff Cristian, this helped me debug an issue I was having with deleting vehicle by id.